Gopkg.toml

```diff
@@ -28,3 +28,12 @@
 [[constraint]]
   branch = "master"
   name = "golang.org/x/net"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/kubevirt/device-plugin-manager"
```

Comment:
isn't "master" the latest commit?

Reply:
I mean latest commit right now, set commit ID, not sure if it would be better since we keep vendor dir

Reply:
do you want me to change the dependency to branch "release-1.9"? this will force k8s of 1.9 as well?
don't think it makes sense to point to latest commit. maybe create a 1.10 brancg on the dpm, and will point there?

Reply:
Thinking again about it, it's not a big deal with vendoring and lock file.